Loop TodayExtension/Base.lproj/MainInterface.storyboard

```diff
@@ -0,0 +1,334 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Per my comment on the issue, ideally these views aren't duplicated. You can create a new framework which contains a nib with these heirarchies and the view classes and share it between targets. This will allow use in future extension targets like notification content.